lms/djangoapps/completion_api/tests/test_views.py

```diff
@@ -204,3 +204,80 @@ def test_staff_access(self):
         self.client.force_authenticate(user)
         response = self.client.get('/api/completion/v0/course/?user={}'.format(self.test_user.username))
         self.assertEqual(response.status_code, 200)
+
+
+@override_settings(STUDENT_GRADEBOOK=True)
+class CompletionBlockUpdateView(SharedModuleStoreTestCase):
+    """
+    Test that the CompletionView renders completion data properly.
+
+    Ensure that it handles authorization as well.
+    """
+
+    usage_key = 'i4x://edX/toy/video/sample_video'
+    incorrect_usage_key = 'i4x://edX/toy/video/another'
+
+    @classmethod
+    def setUpClass(cls):
+        super(CompletionBlockUpdateView, cls).setUpClass()
+        cls.course = ToyCourseFactory.create()
+        cls.another_course = ToyCourseFactory.create()
+
+    def setUp(self):
+        super(CompletionBlockUpdateView, self).setUp()
+        self.test_user = UserFactory.create()
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.test_user)
+        self.update_url = '/api/completion/v0/course/{}/blocks/{}'.format(self.course.id, self.usage_key)
```

Comment:
I'm surprised this works without a trailing slash, since it seemed to be required when I was manually testing.